kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowLayout.kt

```diff
@@ -35,8 +36,16 @@ class WorkflowLayout(
   context: Context,
   attributeSet: AttributeSet? = null
 ) : FrameLayout(context, attributeSet) {
+  private val showing: WorkflowViewStub = WorkflowViewStub(context).also {
+    it.layoutParams = (ViewGroup.LayoutParams(
+        ViewGroup.LayoutParams.MATCH_PARENT,
+        ViewGroup.LayoutParams.MATCH_PARENT
+    ))
```

Comment:
`addView` takes a `LayoutParams` parameter, why do you have to set it explicitly here?

Reply:
Oh yeah! Because I didn't notice that until today. Will update everywhere.